FrontEnd/src/Categories.jsx

```diff
@@ -9,3 +19,28 @@ export const CATEGORIES = [
   "Pet Sitting",
   "Chef",
 ];
+
+const fetchCategoryNames = async() =>{
+  const categoryDocRef = collection(database, POST_CATEGORIES_FOLDER_NAME);
+
+  try{
+    const querySnapshot = await getDocs(categoryDocRef);
+    const newCategories = [...CATEGORIES]
+    if (querySnapshot.empty) {
+      return CATEGORIES;
+    }
+
+    querySnapshot.forEach((doc) => {
+      const categoryName = doc.id;
+      if (newCategories.includes(categoryName) === false) {
+        newCategories.push(categoryName);
+      }
+    });
+    return newCategories;
+  } catch (error) {
+    return CATEGORIES;
```

Comment:
Why still return CATEGORIES if some error happened?

Reply:
There is a default list, I created above, if it cannot fetch more from the database, it should return the original list